test/EnvironmentManagerTests/EnvironmentManagerCentennialTests.h

```diff
@@ -7,7 +7,7 @@ namespace ProjectReunionEnvironmentManagerTests
 {
     class EnvironmentManagerCentennialTests {
         BEGIN_TEST_CLASS(EnvironmentManagerCentennialTests)
-            TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
+            TEST_CLASS_PROPERTY(L"RunAs", L"{UAP,Restricted,Elevated,InteractiveUser}")
```

Comment:
That's new to me. What's it do - run the test once for each context?

Reply:
Close.  This will make one instance of the test class for each entry called metadata[1-4]

For example, Centennial tests have
ProjectReunionEnvironmentManagerTests::EnvironmentManagerCentennialTests#metadataSet[0-3]